content/en/blog/2022/end-user-discussion-group-apac/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "End User Discussion Group: New APAC Sessions Announcement!"
+linkTitle: APAC End User Discussion Group
+date: 2022-12-01
+author: "[Reese Lee](https://github.com/reese-lee)"
+draft: true 
+---
+
+Since July, end users have been getting together to discuss their OpenTelemetry 
+adoption and implementation practices in a vendor-netural space known as the 
+[Monthly Discussion Groups](https://opentelemetry.io/community/end-user/discussion-group/) 
+(also referenced as the End User Discussion Groups). 
+
+Previously, they were only available in the EMEA and AMER regions; the End User 
+Working Group is pleased to announce that **APAC sessions will now be available, 
+starting in January 2023**! 
+
+## The what
+As mentioned above, these Discussion Groups are monthly meetings for end users 
+to discuss the challenges they're facing in their OTel implementations and to 
+learn from each other, as well as to talk about the project itself and what is 
+working, what isn't, and what they'd like to see. 
+
+We encourage you to attend and ask questions to learn how other users have 
+implemented OpenTelemetry in their organizations, resolved common issues, and 
+more. Example topics that have been discussed include tail sampling, collector 
+scaling, and OpAMP; there really is no limit to the topics! We simply ask that 
+you be respectful and abide by the [Chatham House Rule](https://www.chathamhouse.org/about-us/chatham-house-rule#:~:text=The%20Rule%20reads%20as%20follows,other%20participant%2C%20may%20be%20revealed.).
+
+## The why
+This group started as a result of direct feedback from end users in the community! 
+Past participants have shared with us that these have been helpful and valuable 
+for them as they migrate their observability instrumentation to OpenTelemetry. 
+
+Additionally, the End User Working Group is also improving the process to share 
+any collected user feedback* with the appropriate project maintainers to help 
```

Comment:
you might need to escapte that star and the one below otherwise markdown might start to print everything bold from here on (or trip otherwise)